tests/providers/plexus/operators/test_plexus.py

```diff
@@ -0,0 +1,189 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mock
+import pytest
+from mock import Mock
+from requests.exceptions import Timeout
+from airflow.exceptions import AirflowException
+from airflow.providers.plexus.operators.job import PlexusJobOperator
+
+
+class TestPlexusOperator:
+    @pytest.fixture
```

Comment:
Nice! We do not use pytest fixtures too much (we allow both unit tests style and pytest ones). But I think it's good to have more examples of nice pytest fixtures.